### DISCUSSION TITLE
Auto replace inside the editor

### DISCUSSION THREAD

**Discussion by davideio87** [dbeaver/dbeaver#9574] [Q&A] [answered]
Hello,
I didn't find any auto replace function like others IDE (i.e. Toad for Oracle)
It is very handy to assign a shortcut for the most common part of code, i.e. when you type "ss" (or any other short combination of characters, customisable by the user) the editor will write "SELECT * FROM", when you write "oo" it will write "ORDER BY"

This way you can save tons of time, am I the only one this lazy? :smile: 

It is something that is in the roadmap?

Thanks

**Comment by p91paul** [comment]
look for "templates" in preferences. There are some predefined ones and you can add your own. You write the shortcut and then press tab to expand it!

**Comment by kseniiaguzeeva** [comment, marked as answer]
@p91paul are right. Please go to Window>Preferences>DBeaver>Editors>SQL Editor>Templates
![dbeaver_FHbGak6Zl6](https://user-images.githubusercontent.com/51405061/90610467-1cdebb80-e20e-11ea-995a-0634163ceb99.png)
Here is wiki article https://github.com/dbeaver/dbeaver/wiki/SQL-Templates.
Feel free to ask any other questions.
